src/frontend/komponenter/Fagsak/TilkjentYtelse/tilkjentytelse.less

```diff
@@ -1,7 +1,81 @@
+.tilkjentytelse {
+  & .typo-normal:first-child {
```

Comment:
Tror man kan legge begge pseudo-selectorene under typo-normal sånn her: 

  & .typo-normal {
     : first-child {
margin-top: .5rem;
}
:not(:first-child) {
}
  }